module/Product/src/model/Product.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: camelcase
+ * Date: 7/3/18
+ * Time: 10:45 AM
+ */
+
+namespace Product\model;
```

Comment:
Always use TitleCase for namespaces / classnames (`Product\Model` not `Product\model`).